### ISSUE TITLE
Software Design: Use Case Diagram - Teacher

### ISSUE THREAD

**Issue by harunerkurt** [bounswe/bounswe2022group1#130]
**Issue Description**
As part of designing the use case diagram, 3 main user based action sets have been defined. This task corresponds to the design of actions related to teacher.

**Step Details**
Defining the actions that the teacher can make on the platform.
Checking the requirements and adding corresponding parts of lecturer.
Checking the scenarios.
Creating the corresponding diagrams.

**Deadline of the Issue**
10.04.2022 14.00

**Reviewer**
[Kadir Gökhan Sezer](https://github.com/kadirgokhann/)

**Deadline for the Review**
15.04.2022 15.00

**Comment by harunerkurt** [comment]
I defined the actions that the teacher can make on the platform.
I checked the requirements using the class diagram.
I created the template for lecturer.
I created the diagrams related to lecturer.
Diagrams created by me
-  Manage Learning -> Manage Polls, Create Course, Manage Course
<img width="1034" alt="Ekran Resmi 2022-04-14 02 05 34" src="https://user-images.githubusercontent.com/49256548/163284102-4aa3e2c3-43b3-43b9-98b6-b3a667b7be7f.png">
<img width="691" alt="Ekran Resmi 2022-04-14 02 07 55" src="https://user-images.githubusercontent.com/49256548/163284207-c2fe6178-d79d-4511-847f-873dd9238b72.png">
<img width="722" alt="Ekran Resmi 2022-04-14 02 08 21" src="https://user-images.githubusercontent.com/49256548/163284228-c995533e-17d5-4c5d-a566-8a32c0219ae4.png">

Note: in Manage course "edit event" and sub sections of "edit evet" created by Kadir and me.
